findoutlie/metrics.py

```diff
@@ -54,7 +54,42 @@ def dvars(img):
     # You may be be able to solve this in four lines, without a loop.
     # But solve it any way you can.
     data = img.get_fdata()
-    voxel_per_time = np.reshape(data, (-1, data.shape[-1]))
+    voxel_per_time = np.reshape(data, (-1, data.shape[-1])) #np.reshape(data,new_shape)
```

Comment:
```suggestion
    voxel_per_time = data.reshape(-1, data.shape[-1]) #np.reshape(data,new_shape)
```
If you wish for a more compact synthax